src/pages/Pool/v2.tsx

```diff
@@ -160,7 +160,7 @@ export default function Pool() {
                   </ThemedText.White>
                 </RowBetween>
                 <ExternalLink
-                  style={{ color: 'white', textDecoration: 'underline' }}
+                  style={{ color: 'deprecated_white', textDecoration: 'underline' }}
```

Comment:
revert

Reply:
good catch, accidentally did this in one other place but missed these ones. should I update these to use `theme.deprecated_white` though?

Reply:
that works